pyflakes/test/test_undefined_names.py

```diff
@@ -335,6 +335,21 @@ def c(): bar
         def b(): global bar; bar = 1
         ''')
 
+    def test_globalIsUndefinedIfNeverGivenValue(self):
+        """
+        If a "global" is never given a value, it is undefined
+        """
+        self.flakes('''
```

Comment:
I don't think this is necessarily true, a function later in the module could modify that variable and give it a value